plugins/notifications/http/notifier.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3" ; 
```

Comment:
As this file is already present in crowdsec itself, I guess we do not want to duplicate it ?

Reply:
fixed via https://github.com/crowdsecurity/crowdsec/pull/857/commits/80e040abb7f9c2ca6661bf5568742dd0f09076f1